packages/components/src/components/Select/Combobox.tsx

```diff
@@ -0,0 +1,164 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconCaretDown } from '@frontify/fondue-icons';
+import * as RadixPopover from '@radix-ui/react-popover';
+import { Slot as RadixSlot } from '@radix-ui/react-slot';
+import { useCombobox } from 'downshift';
+import { forwardRef, useMemo, useRef, type ForwardedRef, type ReactNode } from 'react';
+
+import { SelectMenu } from './SelectMenu';
+import styles from './styles/select.module.scss';
+import { useSelectData, type SelectItemType } from './useSelectData';
+
+export type ComboboxProps = {
+    /**
+     * Children of the Combobox component. This can contain the `Select.Slot` components for the label, decorators, clear action and menu.
+     */
+    children?: ReactNode;
+    /**
+     * Callback function that is called when an item is selected.
+     */
+    onSelect?: (selectedItem: SelectItemType) => void;
+    /**
+     * The active item in the combobox component. This is used to control the combobox externally.
+     */
+    activeItem?: SelectItemType;
```

Comment:
`defaultValue` => `value` (`string` too) so we stay consistent?

Reply:
changed